recipes/starshot_accessibility/recipe.yml

```diff
@@ -0,0 +1,24 @@
+name: Accessibility
+type: Starshot
+description: 'Sets up accessibility tools and functionality.'
+recipes:
+  - starshot_admin_theme
+install:
+  - a11y_form_helpers
+  - block_aria_landmark_roles
+  - editoria11y
+config:
+  actions:
+    gin.settings:
+      simple_config_update:
+        high_contrast_mode: true
+        accent_color: ''
+        focus_color: ''
```

Comment:
I'd actually suggest moving these Gin settings directly into `starshot_admin_theme`, and dropping it from the `recipes` list here. If these are better for accessibility, I see no reason for them not to be the defaults.

Reply:
Moved to admin_theme recipe